### ISSUE TITLE
image: alpha channel not correctly displayed

### ISSUE THREAD

**Issue by meijeru** [red/red#2812]
In tuple representation, the fourth element is 255 - A, that is understood (default 0 means opaque, hence 255).
However, in binary representation, the last byte of `/argb` differs from the byte of `/alpha` see below. 
```
>> i: make image! 1x1
== make image! [1x1 #{FFFFFF}]
>> i/1
== 255.255.255.0
>> i/rgb
== #{FFFFFF}
>> i/argb
== #{FFFFFFFF}
>> i/alpha
== #{00}  ;;; should be #{FF}
```

**Comment by qtxie** [comment]
Yes, you're right. The alpha channel specify the `Opacity` in ARGB color.

**Comment by meijeru** [comment]
I think the tuple should still have 4th element 255 - A, since the default with 3-element tuples is 4th element = 0, and the default for A is 255. Thus 255 - A, as currently for tuples, is OK.

**Comment by qtxie** [comment]
> Yes, you're right. The alpha channel specify the `Opacity` in ARGB color. So
> 
> * `i/1` should give `255.255.255.255`.
> * ` i/argb` should give `#{FFFFFFFF}`.
> * `i/alpha` should give `#{FF}`.

I think we should do this change. I don't see a good reason to make it different in Red than the rest of the world. As @hiiamboris points out, 

> The alpha component specifies the transparency of the color: 0 is fully transparent, and 255 is fully opaque.

It's same definition on all the platforms we support, also the same on Web CSS.
